### PR TITLE
Fix .gitignore based on ikesyo's comment

### PR DESCRIPTION
**Pull request by skywalkerdude**
Hopefully this makes the build faster. Closes https://github.com/HymnalDreamTeam/Hymns-iOS/issues/152

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -67,8 +67,8 @@ Pods/*
 Carthage/Build/
 
 # Accio dependency management
-Dependencies/
-.accio/
+# Dependencies/
```
Comment (ikesyo) [inline .gitignore:70]: Looks like now that you ignore entire `Pods` dir (since #66) so this is meaningless.

Comment (skywalkerdude) [reply .gitignore:70]: @ikesyo, I ignore the entire Pods directory but I also do a `pod install` during the build step which will pull in all the pods. I can remove this and see if a simple pod install will work, but my impression was that only doing pod install will cause the build issue (https://github.com/Quick/Nimble/issues/490). I can revert this change and see if it still the case

Comment (skywalkerdude) [reply .gitignore]: ah, looks like it passes without the pod deintegrate. great!

Comment (ikesyo) [reply .gitignore]: > it passes without the pod deintegrate

Yes, that is the poiint.

**Pull request merged**